airbyte-integrations/connectors/source-mongodb-v2/README.md

```diff
@@ -1,5 +1,27 @@
 # MongoDb Source
 
+## Schema
+
+For each collection, if you want to define a schema in order to propagate type to Glue, specify the schema for the collection as a unique string. 
+
+Example:
+
+embedded_movies:{"fields":[{"name":"_id"COMMA"type":"string"}COMMA{"name":"tomatoes"COMMA"type":"object"COMMA"subfields":[{"name":"production"COMMA"type":"string"}]}]}
```

Comment:
What is COMMA?

Reply:
I'll see if I can remove this by changing the spec json